sdk/js/tsconfig.spec.json

```diff
@@ -4,11 +4,5 @@
         "outDir": "../../dist/out-tsc",
         "types": ["jest", "node"]
     },
-    "include": [
-        "**/*.test.ts",
-        "**/*.spec.ts",
-        "**/*.d.ts",
-        "jest.config.ts",
-        "**/__mocks__/*.ts"
-    ]
+    "include": ["**/*.test.ts", "**/*.spec.ts", "**/*.d.ts", "jest.config.ts"]
```

Comment:
why remove the mocks entry?

Reply:
It was complaining about the fetch mock I think. MIght not be necessary anymore, will double check

Reply:
removing it would likely give a type error if trying to use `jest` anywhere in a mocks file